karma.conf.js

```diff
@@ -14,7 +14,7 @@ module.exports = function (config) {
     preprocessors: {
```

Comment:
`onJenkins` is still used to decide whether generate `html` or `lcov` report.  
I removed the `karma-coveralls`, dependency.